packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -118,24 +119,26 @@ export const downloadDirectoryAction: GeneratorAction = {
 };
 
 export const fetchUrlForHotfixOnlyAction: GeneratorAction = {
-  name: GeneratorActionName.FetchTemplateUrlWithTag,
+  name: GeneratorActionName.FetchUrlForHotfixOnly,
   run: async (context: GeneratorContext) => {
-    if (context.zip || context.url || context.cancelDownloading) {
+    if (context.zip || context.cancelDownloading) {
       return;
     }
 
     context.logProvider.debug(`Fetching template url with tag: ${JSON.stringify(context)}`);
-    const url = await fetchTemplateZipUrl(
+    const latestVer = await getTemplateLatestVersion(
```

Comment:
This function returns a tag, not a function. So you either change the function name or update it to return a version.

Reply:
I'll change the function name.